frontend/src/components/search/SearchResultsListItem.tsx

```diff
@@ -0,0 +1,129 @@
+"use client";
+
+import { Opportunity } from "src/types/search/searchResponseTypes";
+import { formatDate } from "../../utils/dateUtil";
+
+interface SearchResultsListItemProps {
+  opportunity: Opportunity;
+}
+
+export default function SearchResultsListItem({
+  opportunity,
+}: SearchResultsListItemProps) {
+  const opportunityURL =
+    process.env.NEXT_PUBLIC_ENVIRONMENT === "prod"
+      ? "https://grants.gov"
+      : "https://test.grants.gov";
```

Comment:
we'll have to double check this when we're able to deploy to lower envs . I think it's only a nit for me to add a TODO with a "relates to" this ticket https://github.com/HHS/simpler-grants-gov/issues/1521